src/main/java/frc/robot/subsystems/Motion.java

```diff
@@ -84,9 +84,13 @@ private void setWristPivotAngle(Angle angle) {
   }
 
   public void setAllPosition(MechanismPositionGroup positionGroup) {
-    setLiftPosition(positionGroup.liftHeight);
-    setElevatorPivotAngle(positionGroup.pivotAngle);
-    setWristPivotAngle(positionGroup.wristAngle);
+    if (isWristInDanger() == true) {
+      setWristPivotAngle(positionGroup.wristAngle);
+    } else {
+      setLiftPosition(positionGroup.liftHeight);
+      setElevatorPivotAngle(positionGroup.pivotAngle);
+      setWristPivotAngle(positionGroup.wristAngle);
```

Comment:
If wrist is in danger move to right outside of danger

Otherwise move to wrist angle

The other mechanisms need to be outside of else statement so they keep moving

Reply:
@Wu-Fan-529 or @BrodyKarr can either of you fix